src/providers/json-rpc-provider.ts

```diff
@@ -98,10 +98,22 @@ export class JsonRpcProvider extends Provider {
      * @param txHash The hash of the transaction
      * @param accountId The NEAR account that signed the transaction
      */
-    async txStatus(txHash: Uint8Array, accountId: string): Promise<FinalExecutionOutcome> {
+    async txStatus(txHash: any, accountId: string): Promise<FinalExecutionOutcome> {
```

Comment:
nit: why not `Uint8Array | string`?

also need to document what format is expected for string (is it base64 or base58?)

Reply:
Great point. I'll update